docs/src/styles/docs.css

```
@@ -1,463 +1,486 @@
+  background: color-mix(in srgb, var(--sl-color-accent) 10%, transparent);
+  color: var(--sl-color-accent);
+  font-weight: 500;
 }
-
-/* Tighten code blocks inside cards */
-.hero ~ .sl-markdown-content .card .expressive-code {
-  margin-top: 0.75rem;
+.sidebar-content a[aria-current='page'] .sl-badge {
+  --sl-color-bg-badge: transparent;
+  --sl-color-border-badge: color-mix(in srgb, currentColor 34%, transparent);
+  color: inherit;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)